### ISSUE TITLE
Remove direct database connection with GROWTH marshal

### ISSUE THREAD

**Issue by lpsinger** [growth-astro/growth-too-marshal#67]
We are going to be moving to AWS soon. Maintaining a direct connection to the GROWTH marshal database will be complicated. We do not use the connection for much right now anyway; there are just these two tables that we have bound:

```python
class Users(models.db.Model):
    __bind_key__ = 'growthdb'
    __table__ = Table(
        'users', models.db.metadata, autoload=True,
        autoload_with=models.db.get_engine(app, bind=__bind_key__))


class SciencePrograms(models.db.Model):
    __bind_key__ = 'growthdb'
    __table__ = Table(
        'scienceprograms', models.db.metadata, autoload=True,
        autoload_with=models.db.get_engine(app, bind=__bind_key__))
```

Can we remove the direct connection to the GROWTH postgres database?

**Comment by mcoughlin** [comment]
@lpsinger @igorandreoni I can help with this, but I think Igor is best placed to comment on how to deal with this.

**Comment by igorandreoni** [comment]
I am quite sure that those two tables are not in use at the moment (they were there before I joined the team).  In any case, we were required to remove any direct connection to the GROWTH db.

**Comment by lpsinger** [comment]
Closed by #68.
